site/app/templates/polls/ViewPoll.twig

```diff
@@ -22,34 +22,35 @@
                 <input type="hidden" name="csrf_token" value="{{ csrf_token }}"/>
                 <input type="hidden" name="poll_id" value="{{ poll.getId() }}"/>
                 <fieldset>
+                    <table>
                     <legend>
                         <h2>
                             Possible responses:
                         </h2>
                     </legend>
                     {% set button_type = is_single_response ? "radio": "checkbox" %}
                     {% if not user_admin %}
-                        <label class="radio" for="answer-no-response">
-                            <input class="no-response-radio response-btn"
+                        <tr class="radio" for="answer-no-response">
+                            <td><input class="no-response-radio response-btn"
                                    type="{{button_type}}"
                                    value="-1"
                                    name="answers[]"
                                    id="answer-no-response"
                                    {{ poll.getUserResponses()|length == 0 ? "checked" : "" }}
                                    {{ (not user_admin and poll.isOpen()) ? "" : "disabled" }}
                                    onchange="clearResponses()"
-                            >
-                            No response
-                        </label>
-                        <br/>
-                        <br/>
+                            /></td>
+                            <td class="no-response option-text">
+                            no response
+                            </td>
+                        </tr>
```

Comment:
I think this should still read "No response".  It's common in English to capitalize the first letter of a phrase or sentence such as in this case.

Reply:
More of an opinion thing, this definitely is not a complete sentence.
But, yes, chief.

Reply:
I'm going to abandoned the styling on "No response" if the first letter is capitalized - since now its distracting.